### DISCUSSION TITLE
Override views - bug or regression in the documentation ?

### DISCUSSION THREAD

**Discussion by lamoureuxmarie** [solidusio/solidus#4651] [Troubleshooting] [answered]
Hi all 👋

I'm just starting with Solidus, and I'm a bit confused on how to customize it. 

If I understand well, I need to create my own frontend views in a new folder.
I tried this command to generate a copy of the views:

`bundle exec rails g solidus:views:override`

It gives me this error message: `Could not find generator 'solidus:views:override`

I created the app with the following command:

```
rails new \
  -d postgresql \
  -j webpack \ solidus-test
bundle add solidus
bin/rails g solidus:install
```

When I asked in the support group on Slack, I have been informed that it looks like a bug or regression in the documentation.

(I'm using Rails 7.)

**Comment by elia** [comment, marked as answer]
I guess this is coming from [this page](https://guides.solidus.io/developers/customizations/customizing-storefront#customizing-views) in the guides, which is outdated. The new guides are available at https://edgeguides.solidus.io and the corresponding page there seems to be https://edgeguides.solidus.io/customization/customizing-your-storefront/. I think we're about to switch the domains so those  will become official shortly, so I don't think it's worth trying to fix them.

What happens is that the original guide was assuming the frontend was [solidus_frontend](https://github.com/solidusio/solidus_frontend) while new installation will use  [solidus_starter_frontend](https://github.com/solidusio/solidus_starter_frontend) by default, copying all views and controllers directly into the rails app, so you should already have all you need to start customizing.

**Comment by lamoureuxmarie** [comment]
Thank you, Elia, for these explanations! Very helpful ☺️
